### PR TITLE
cli: Fix up usage message

### PR DESCRIPTION
**Pull request by jsoref**
The properties file naively assumed that leading spaces would be honored.
-- They are not. To get some indentation, one needs to escape a space first.

Further, for no particular reason, the last paragraph had a hard wrap.

Finally, since this commit has to reindent the entire block, fix the alignment of the `:`s
for most of the options.
(-auth is a mess and should be rewritten)
### Proposed changelog entries

* clean up whitespace for cli usage message

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

File: cli/src/main/resources/hudson/cli/client/Messages.properties [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+  \ -user               : specify user (for use with -ssh)\n\
+  \ -strictHostKey      : request strict host key checking (for use with -ssh)\n\
+  \ -logger FINE        : enable detailed logging from the client\n\
+  \ -auth [ USER:SECRET | @FILE ] : specify username and either password or API token (or load from them both from a file);\n\
+  \                                 for use with -http, or -remoting but only when the JNLP agent port is disabled.\n\
+  \                                 Passing crendentials by a file is recommended.\n\
+  \                                 See https://jenkins.io/redirect/cli-http-connection-mode for more info and options.\n\
   \n\
-  The available commands depend on the server. Run the 'help' command to\n\
+  The available commands depend on the server. Run the 'help' command to \
```
Comment (oleg-nenashev) [inline cli/src/main/resources/hudson/cli/client/Messages.properties]: Hmm. why?

**Comment by oleg-nenashev** [comment]
@jsoref hopefully I resolved the merge conflict correctly

**Review by oleg-nenashev — APPROVED** [review]

File: cli/src/main/resources/hudson/cli/client/Messages.properties (hunk 1 above)
Comment (jsoref) [reply cli/src/main/resources/hudson/cli/client/Messages.properties]: It's an arbitrary word wrapping location. Other lines above are wider.

**Comment by jsoref** [comment]
I think the merge looks correct.

**Pull request merged**